app/views/profiles/_attendances_details.html.erb

```diff
@@ -11,11 +11,17 @@
     </div>
 
     <ul class="<%= status %> attendees-list">
-      <% attendances.map(&:user).each do |user| %>
+      <% attendances.each do |attendance| %>
+        <% user = attendance.user %>
         <li class="attendee-names">
           <%= link_to user.name, "mailto:#{user.email}", :target => "_blank" %>
+          <% if ShowAttendeePhoneNumberForUser.new(current_user, attendance).call() %>
```

Comment:
Usually `.call` is also a class method also takes all the arguments rather than having to be instantiated.